nengo_spa/algebras/tests/test_algebras.py

```diff
@@ -52,7 +52,7 @@ def test_binding_and_invert(algebra, d, sidedness, rng):
                 warnings.simplefilter("error", DeprecationWarning)
                 if binding_side is ElementSidedness.LEFT:
                     bound = algebra.bind(b, a)
-                    r = algebra.bind(bound, algebra.invert(b, sidedness=sidedness))
+                    r = algebra.bind(algebra.invert(b, sidedness=sidedness), bound)
```

Comment:
I'm guessing this was an error that wasn't evident until TVTB and it's sidedness was implemented?

Reply:
Yes, the left-inverse needs to be applied from the left.